test/test.spec.tsx

```diff
@@ -29,6 +29,23 @@ describe("Simple Styled", () => {
     });
   });
 
+  test("Creates input properly", () => {
+    const Input = styled('input')`
```

Comment:
I might be wrong, but the issue was pointing out that we should handle the case of something like:
```
const SpecialInput = styled(Input)``
```
Isn't worth to add a test case where you test this too?